blocks/test/full-content.js

```diff
@@ -127,7 +127,7 @@ describe( 'full post content fixture', () => {
 				}
 			}
 
-			expect( serializedActual ).to.eql( serializedExpected );
+			expect( serializedActual.trim() ).to.eql( serializedExpected.trim() );
```

Comment:
In my opinion we shouldn't be testing for the specific presence of absence of it.